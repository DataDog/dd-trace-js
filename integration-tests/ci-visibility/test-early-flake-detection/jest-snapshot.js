describe('test', () => {
  it('can do snapshot', () => {
    // eslint-disable-next-line
    expect(1 + 2).toMatchSnapshot()
  })
})
