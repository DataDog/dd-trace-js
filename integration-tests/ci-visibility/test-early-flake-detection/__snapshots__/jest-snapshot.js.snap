// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test can do snapshot 1`] = `3`;
